__tests__/endpoints.test.js

```diff
@@ -98,6 +98,32 @@ describe('GET /api/articles', () => {
         expect(sortedArticles).toBeSortedBy('created_at', { descending: true });
       });
   });
+
+  it('should respond with articles sorted by votes in ascending order when passed a sort_by query', () => {
+    return request(app)
+      .get('/api/articles?sort_by=votes&order=asc') 
+      .expect(200)
+      .then(({ body }) => {
+        const sortedArticles = body.articles.map(article => ({
+          ...article,
+          votes: article.votes,
+        }));
+        expect(sortedArticles).toBeSortedBy('votes', { descending: false });
+      });
+  });
+
+  it('200 - responds with articles sorted by created_at in ascending order when passed sort_by and order', () => {
+    return request(app)
+      .get('/api/articles?sort_by=created_at&order=asc')
+      .expect(200)
+      .then(({ body }) => {
+        const sortedArticles = body.articles.map(article => ({
+          ...article,
+          created_at: new Date(article.created_at),
+        }));
+        expect(sortedArticles).toBeSortedBy('created_at', { descending: false });
+      });
```

Comment:
You don't need to spread the articles into a new object for this test, as you can directly check the sorted order 👍